sdk/test/Services/S3/UnitTests/Custom/ResponseMapperTests.cs

```diff
@@ -165,7 +165,7 @@ public void MapUploadRequest_PutObjectRequest_AllMappedProperties_WorkCorrectly(
                     var simpleUploadCommand = new SimpleUploadCommand(null, null, sourceRequest);
                     return simpleUploadCommand.ConstructRequest();
                 },
-                usesHeadersCollection: false,
+                usesHeadersCollection: true,
```

Comment:
since CacheControl was added to mapping and thats located in headers we have to search there now